third_party/webcomponentsjs/ShadowCSS.js

```diff
@@ -1,11 +1,9 @@
 /**
  * @license
  * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
- * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
- * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
- * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
- * Code distributed by Google as part of the polymer project is also
- * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
```

Comment:
thanks for this 👍

Reply:
@erwinmombay This change is likely a good idea outside of this PR. Want to make it and save everyone some bytes?

Reply:
@kristoferbaxter do you mind clarifying what change? just changing the licenses in the external deps?